AppCenter/AppCenter/Internals/Util/MSPerformSelectorUtil.h

```diff
@@ -3,9 +3,24 @@
 
 #import <Foundation/Foundation.h>
 
+#define MS_DISPATCH_SELECTOR_OBJECT(type, object, selectorName, ...)                                                                       \
+  ({                                                                                                                                       \
+    void *results;                                                                                                                         \
+    [[MSPerformSelectorUtil performSelector:object withSelector:@ #selectorName withObjects:@[ __VA_ARGS__ ]] getReturnValue:&results];    \
+    (__bridge type) results;                                                                                                               \
```

Comment:
No need a separate define for just one `__bridge` word. It can be passed to params

Reply:
Fixed